README.md

```diff
@@ -1,7 +1,18 @@
 # synapseformation
 Client for using Synapse Formation templates.
 
+## Usage
+```
+import synapseclient
+from synapseformation import create
+
+syn = synapseclient.login()
+# Only create entities
+CreateCls = create.SynapseCreation(syn)
+# Only retrieve entities (don't update)
+RetrieveCls = create.SynapseCreation(syn, only_create=False)
```

Comment:
Hmmm, that is an interesting suggestion.  I'm honestly uncertain what it _should_ be.  There are several issues currently:

* Doesn't allow for any updating of entities (only creating or retrieving)
* The code exists ungracefully when the `only_create` is false but the entity doesn't exist (_I think_)

`dry_run` to me sometimes implies that a Synapse rest call isn't made, but in this case, entities are retrieved if they exist.  Thoughts?

Reply:
My bad, I misunderstood what the argument was doing. I agree that `dry_run` wouldn't be appropriate based on your description, and `only_create` is fine for now. 

As an aside, what use case do you have in mind for an argument like `only_create` where the user strictly only wants to create entities and otherwise fail if the entity already exists? In my mind, the opposite would be more common, where a user wants to avoid creating/modifying anything (_i.e._ an error would occur if an entity doesn't exist). Something like a `allow_creation=False` argument, although the default value can be whatever makes sense. I'm suggesting `False` here to err on the safe side.

Reply:
Thanks @BrunoGrandePhD.  Hmmm.  I think because the package is synapseformation, there is the aspect of "creating" entities, which is why i think it would be a little funny for a formation client to have an `allow_creation` flag.  By default `only_create` is True, but maybe it should be `only_get=False` or `only_retrieve=False` instead?

Reply:
`only_get=False` makes sense to me. That said, I don't think `only_create` is unintuitive now that I understand what it does, so changing the argument seems less of a priority, if at all needed.